### PR TITLE
faq, sql: Document tidb_force_priority

### LINKED ISSUE #629: Document tidb_force_priority = low_priority
References:
https://github.com/pingcap/tidb/issues/7524
https://github.com/pingcap/tidb/pull/7694
https://github.com/pingcap/tidb/pull/7777

It should be on this manual page: https://pingcap.com/docs/sql/server-command-option/

And possibly in a section under "Advanced Usage" recommending strategies to improve QoS of query time.

### PR DESCRIPTION
**Pull request by morgo**
Fixes #629.

### CODE REVIEW

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -282,6 +282,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
 - You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
 
+### tidb_force_priority
+
+- Scope: SESSION | GLOBAL
```
Comment (jackysp) [inline sql/tidb-specific.md]: It is a session-only variable though it works on the server level. `GLOBAL` variable will work on all TiDB servers.

File: sql/tidb-specific.md [hunk 2]
```diff
@@ -282,6 +282,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
 - You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
 
+### tidb_force_priority
+
+- Scope: SESSION
+- Default value: `NO_PRIORITY`
+- This variable is used to change the default priority for statements executed on a TiDB server.  A use case is to ensure that a particular user that is performing OLAP queries receives lower priority than users performing OLTP queries.
+- You can set the value of this variable to `PRIORITY_LOW`, `NO_PRIORITY` or `PRIORITY_HIGH`.
```
Comment (jackysp) [inline sql/tidb-specific.md]: It's better to use
```
`NO_PRIORITY`, `LOW_PRIORITY`, `HIGH_PRIORITY` or `DELAYED`
```
tidb_force_priority uses SQL priority options as its available values which is not the same as tidb_ddl_reorg_priority does.

File: sql/server-command-option.md [hunk 3]
```diff
@@ -157,6 +157,14 @@ Same as the "run-ddl" startup option
 - Default: true
 - When you execute `join` on tables without any conditions on both sides, the statement can be run by default. But if you set the value to `false`, the server does not run such `join` statement. 
 
+### force-priority
+
+- The default priority for statements
+- Default: `NO_PRIORITY`
+- TiDB supports the priorities `LOW_PRIORITY`, `NO_PRIORITY`, `HIGH_PRIORITY` and `DELAYED` for statements.  One use case for changing the priority, is you may choose to dedicated a pool of servers for OLAP queries and set the value to `LOW_PRIORITY` to ensure that TiKV servers will provide priority to OLTP workloads which are routed to a different pool of TiDB servers.  This helps ensure more uniform OLTP performance at the risk of slightly slower OLAP performance.
```
[the remark this answers is not in the document]
Comment (jackysp) [reply sql/server-command-option.md]: It's better to use a union priority order, how about
```
LOW_PRIORITY | DELAYED | HIGH_PRIORITY | NO_PRIORITY
````
or
```
NO_PRIORITY | LOW_PRIORITY | DELAYED | HIGH_PRIORITY
````

**Comment by lilin90** [comment]
@CaitinChen PTAL again

File: sql/server-command-option.md [hunk 4]
```diff
@@ -157,6 +157,14 @@ Same as the "run-ddl" startup option
 - Default: true
 - When you execute `join` on tables without any conditions on both sides, the statement can be run by default. But if you set the value to `false`, the server does not run such `join` statement. 
 
+### force-priority
+
+- The default priority for statements
+- Default: `NO_PRIORITY`
+- TiDB supports the priorities `NO_PRIORITY | LOW_PRIORITY | DELAYED | HIGH_PRIORITY` for statements. One use case for changing the priority, is you may choose to dedicate a pool of servers for OLAP queries and set the value to `LOW_PRIORITY` to ensure that TiKV servers will provide priority to OLTP workloads which are routed to a different pool of TiDB servers. This helps ensure more uniform OLTP performance at the risk of slightly slower OLAP performance.
```
Comment (CaitinChen) [inline sql/server-command-option.md]: `NO_PRIORITY | LOW_PRIORITY | DELAYED | HIGH_PRIORITY` 
->
`NO_PRIORITY` | `LOW_PRIORITY` | `DELAYED` | `HIGH_PRIORITY`

File: sql/server-command-option.md [hunk 5]
```diff
@@ -157,6 +157,14 @@ Same as the "run-ddl" startup option
...
+  
+TIDB will automatically set table scans to `LOW_PRIORITY` and overwriting priority on a per-statement basis is possible by using the `HIGH PRIORITY` or `LOW PRIORITY` [DML modifier](dml.md).
```
Comment (CaitinChen) [inline sql/server-command-option.md]: TIDB -> TiDB

File: sql/tidb-specific.md [hunk 6]
```diff
@@ -282,6 +282,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
 - You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
 
+### tidb_force_priority
+
+- Scope: SESSION
+- Default value: `NO_PRIORITY`
+- This variable is used to change the default priority for statements executed on a TiDB server.  A use case is to ensure that a particular user that is performing OLAP queries receives lower priority than users performing OLTP queries.
```
Comment (CaitinChen) [inline sql/tidb-specific.md]: Please delete a space before "A use case".

**Pull request merged**